boilerplate/_lib/container-make

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -x
+#!/bin/bash -e
```

Comment:
We can't add -e as is. The script isn't set up for it. For example, L22 will bail without giving us a chance to check the RC and print a helpful message. Likewise L30 would bail and not let us start the container for you to debug in.